.github/workflows/release-cpo.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
 
     - name: Get the version from ref
       id: get_version
-      run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+      run: echo VERSION=${GITHUB_REF/refs\/tags\//} >> $GITHUB_OUTPUT
```

Comment:
didn't read the whole file, just curious, we created GITHUB_OUTPUT file ,then where do we use it?

Reply:
@jichenjc Hi there. The old `set-output` has been deprecated and replaced by putting a value in the environment variable file `$GITHUB_OUTPUT`. If you look at line 36 below the echo command, you can see that it uses the value obtained by the echo command, such as `VERSION=${{ steps.get_version.outputs.VERSION }}`.